src/main/java/com/cloudbees/jenkins/support/impl/FileDescriptorLimit.java

```diff
@@ -144,6 +145,7 @@ public String call() {
     /**
      * * Using OperatingSystemMXBean, we can obtain the total number of open file descriptors.
      */
+    @IgnoreJRERequirement
```

Comment:
Ought to be accompanied by a Java version check in `GetUlimit`, or at least catching `LinkageError` and reporting it gracefully.